apps/files_sharing/src/components/SharingEntryLink.vue

```diff
@@ -133,6 +149,14 @@
 						{{ t('files_sharing', 'Customize link') }}
 					</NcActionButton>
 				</template>
+				
+				<NcActionButton :close-after-click="true"
+					@click.prevent="showQRCode = true">
+					<template #icon>
+						<IconQr />
+					</template>
+					{{ t('files_sharing', 'Generate QR Code') }}
```

Comment:
I am not 100% sure here but I think it should not be a capitalized C?
```suggestion
					{{ t('files_sharing', 'Generate QR code') }}
```

Reply:
Agreed! More consistent with the rest of the capitalization of the other actions 👍